src/tests.rs

```diff
@@ -897,6 +902,7 @@ fn _init_trace_logger() {
 fn mock_simple_init() {
     // (Pass) Should look the same as a fresh 'vet init'.
 
+    let _enter = TEST_RUNTIME.enter();
```

Comment:
this is a bit of a nightmare but *shrug*

Reply:
Yeah, I couldn't come up with a better option, so I just did a sed pass basically. I think we might want to have some macro or something like that which we just invoke at the start of a function to do setup perhaps? Would also let us set up tracing in tests for example.

Reply:
there's already a "setup tracing" function in here, I just add it to any test I want to debug at the time

Reply:
That's fair. If our entire stack was async from the root, we could make these all `#[tokio::test]`, but you can't `block_on` inside of a `#[tokio::test]`.